src/command_modules/azure-cli-relay/azure/cli/command_modules/relay/_params.py

```diff
@@ -0,0 +1,142 @@
+
+    for scope in ['relay namespace create', 'relay namespace update']:
+        with self.argument_context(scope) as c:
+            c.argument('tags', arg_type=tags_type)
+            c.argument('sku', arg_type=get_enum_type(SkuTier))
+
+    with self.argument_context('relay namespace create') as c:
+        c.argument('location', arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
+
+    # region Namespace Authorization Rule
+    with self.argument_context('relay namespace authorization-rule list') as c:
+        c.argument('namespace_name', options_list=['--namespace-name'], id_part=None, help='Name of the Namespace')
+
+    with self.argument_context('relay namespace authorization-rule') as c:
+        c.argument('authorization_rule_name', arg_type=name_type, id_part='child_name_1', help='Name of Namespace Authorization Rule')
+        c.argument('namespace_name', id_part='name', options_list=['--namespace-name'], help='Name of Namespace')
+
+    for scope in ['relay namespace authorization-rule create', 'relay namespace authorization-rule update', 'relay wcfrelay authorization-rule create', 'relay wcfrelay authorization-rule update', 'relay hyco authorization-rule create', 'relay hyco authorization-rule update']:
+        with self.argument_context(scope) as c:
+            c.argument('rights', arg_type=rights_arg_type)
+
+    with self.argument_context('relay namespace authorization-rule keys renew') as c:
+        c.argument('key_type', arg_type=key_arg_type)
+        c.argument('key', arg_type=keyvalue_arg_type)
+
+    with self.argument_context('relay namespace authorization-rule keys list') as c:
+        c.argument('authorization_rule_name', arg_type=name_type, id_part=None, help='Name of Namespace Authorization Rule')
+        c.argument('namespace_name', id_part=None, options_list=['--namespace-name'], help='Name of Namespace')
+
+    # region WCF Relay
+    with self.argument_context('relay wcfrelay') as c:
+        c.argument('relay_name', arg_type=name_type, id_part='child_name_1', completer=get_wcfrelay_command_completion_list, help='Name of WCF Relay')
+
+    # region - WCF Relay Create
+    for scope in ['relay wcfrelay show', 'relay wcfrelay delete']:
+        with self.argument_context(scope) as c:
+            c.argument('relay_name', arg_type=name_type, id_part='child_name_1', completer=get_wcfrelay_command_completion_list, help='Name of WCF Relay')
+
+    with self.argument_context('relay wcfrelay create') as c:
+        c.argument('relay_name', arg_type=name_type, id_part='child_name_1', help='Name of WCF Relay')
+        c.argument('status', arg_type=get_enum_type(['Active', 'Disabled', 'SendDisabled', 'ReceiveDisabled']), help='Enumerates the possible values for the status of a messaging entity.')
+        c.argument('relay_type', arg_type=get_enum_type(['Http', 'NetTcp']), help='Relay type, Http or NetTcp (default).')
```

Comment:
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-enums

You can set the default here. `default='NetTcp`. Help should just say "Relay type."